lib/ap/crt/crt1.S

```diff
@@ -6,10 +6,26 @@
 .globl	_start
 _start:
 
+	// 24 bytes room for environ
```

Comment:
Not sure too. In 32 it needed 12. The trick for keeping it is putting it in stack, because through regs (rdx, rbx or others callee) appears to be overriden by exec and rfork derivatives. Sadly I didn't get the right formula yet. Help needed. That's why this PR is WIP.